aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -1063,6 +1063,8 @@ _scaled_mm_out_cuda(const Tensor& mat1, const Tensor& mat2,
     return out;
   }
 
+  // ROCm's hipblaslt supports rowwise, so skip this check that sends this to cutlass.
+#ifndef USE_ROCM
```

Comment:
Hipblaslt supports more than bf16 right?

Reply:
From my local mi300 tests of stock rocm 6.3 version of hipblaslt, only bf16 output is supported for rowwise.  float32 and float16 are producing RuntimeError: could not find valid hipblaslt solution.

Reply:
should we add as a check then?

Reply:
another good suggestion. done.